tc/sgx/trusted_worker_manager/enclave/base_enclave.cpp

```diff
@@ -28,6 +28,7 @@
 #include "auto_handle_sgx.h"
 
 #include "error.h"
+#include "avalon_sgx_error.h"
 #include "tcf_error.h"
```

Comment:
We can combine error.h and tcf_error.h into avalon_error.h. May be in another PR ?

Reply:
Yes, i will combine them in another PR.